base_layer/contacts/src/contacts_service/types/message.rs

```diff
@@ -36,7 +36,8 @@ use crate::contacts_service::proto;
 pub struct Message {
     pub body: Vec<u8>,
     pub metadata: Vec<MessageMetadata>,
-    pub address: TariAddress,
+    pub to_address: TariAddress,
+    pub from_address: TariAddress,
```

Comment:
I think this is more in line with the rest of the naming in the code base.
```suggestion
    pub receiver_address: TariAddress,
    pub sender_address: TariAddress,
```